.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
can these comments be removed? They make this file invalid JSON

Reply:
👍